src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkTask.java

```diff
@@ -56,52 +51,53 @@ public void start(Map<String, String> props) {
   }
 
   // public for testing
-  public void start(Map<String, String> props, JestClient client) {
+  public void start(Map<String, String> props, ElasticsearchClient client) {
     try {
       log.info("Starting ElasticsearchSinkTask.");
 
       ElasticsearchSinkConnectorConfig config = new ElasticsearchSinkConnectorConfig(props);
-      String type = config.getString(ElasticsearchSinkConnectorConfig.TYPE_NAME_CONFIG);
-      boolean ignoreKey =
+      final String type = config.getString(ElasticsearchSinkConnectorConfig.TYPE_NAME_CONFIG);
```

Comment:
Were all these `final`s required by checkstyle? Otherwise it's a lot of noise that doesn't correspond to actual changes. 

Additionally to that, the connectors style does not require to use `final` keyword for local variables.

Reply:
These finals were recommended by checkstyle at one point but I think they are not needed any longer so I can remove.